src/test/resources/org/kohsuke/github/ReleaseTest/wiremock/testCreateSimpleRelease/__files/repos_hub4j-test-org_testcreaterelease_releases-5.json

```diff
@@ -0,0 +1,40 @@
+{
+  "url": "https://api.github.com/repos/hub4j-test-org/testCreateRelease/releases/44426833",
+  "assets_url": "https://api.github.com/repos/hub4j-test-org/testCreateRelease/releases/44426833/assets",
+  "upload_url": "https://uploads.github.com/repos/hub4j-test-org/testCreateRelease/releases/44426833/assets{?name,label}",
+  "html_url": "https://github.com/hub4j-test-org/testCreateRelease/releases/tag/38c0b3e9-2abc-4e79-b791-aeab4cc4efcd",
+  "id": 44426833,
+  "author": {
+    "login": "jlengrand",
+    "id": 921666,
+    "node_id": "MDQ6VXNlcjkyMTY2Ng==",
+    "avatar_url": "https://avatars.githubusercontent.com/u/921666?v=4",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/jlengrand",
+    "html_url": "https://github.com/jlengrand",
+    "followers_url": "https://api.github.com/users/jlengrand/followers",
+    "following_url": "https://api.github.com/users/jlengrand/following{/other_user}",
+    "gists_url": "https://api.github.com/users/jlengrand/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/jlengrand/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/jlengrand/subscriptions",
+    "organizations_url": "https://api.github.com/users/jlengrand/orgs",
+    "repos_url": "https://api.github.com/users/jlengrand/repos",
+    "events_url": "https://api.github.com/users/jlengrand/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/jlengrand/received_events",
+    "type": "User",
+    "site_admin": false
+  },
+  "node_id": "MDc6UmVsZWFzZTQ0NDI2ODMz",
+  "tag_name": "38c0b3e9-2abc-4e79-b791-aeab4cc4efcd",
+  "target_commitish": "main",
+  "name": "release-38c0b3e9-2abc-4e79-b791-aeab4cc4efcd",
+  "draft": false,
+  "prerelease": false,
+  "created_at": "2021-06-02T21:59:14Z",
+  "published_at": "2021-06-10T16:08:28Z",
+  "assets": [],
+  "tarball_url": "https://api.github.com/repos/hub4j-test-org/testCreateRelease/tarball/38c0b3e9-2abc-4e79-b791-aeab4cc4efcd",
+  "zipball_url": "https://api.github.com/repos/hub4j-test-org/testCreateRelease/zipball/38c0b3e9-2abc-4e79-b791-aeab4cc4efcd",
+  "body": null,
+  "discussion_url": "https://github.com/hub4j-test-org/testCreateRelease/discussions/2"
```

Comment:
Interesting point.  There is a `discussion_url` here. Is this always present or only when the category provided?

Reply:
I'll look into it. What would you want to be doing with it?

Reply:
Added the value in ghRelease and added test checking the value of the item when (and when not) setting the category_name value in the releases